src/components/pluralize.cjsx

```diff
@@ -0,0 +1,18 @@
+React = require 'react'
+
+# TODO find a better pluralizer
```

Comment:
I think it was probably underscore.inflections.  It does pluralization and titleize, camelize and a few other useful ones.  It can also be customized for special words that need special pluralization rules

https://github.com/geetarista/underscore.inflections